### PR TITLE
Adapt layout to codex's grid system

### PR DESCRIPTION
**Pull request by chukarave**
This PR adapts the layout to the new Codex values, Mostly breakpoints and max-width values. 

Bug: T348729

### CODE REVIEW

File: resources/js/Components/ItemIdSearchTextarea.vue [hunk 1]
```diff
@@ -98,8 +98,7 @@ defineExpose({validate, serializeInput, validationError});
 </script>
 
 <style lang="scss">
-/* stylelint-disable-next-line scss/load-no-partial-leading-underscore */
-@import '~@wmde/wikit-tokens/dist/_variables';
```
Comment (chukarave) [inline resources/js/Components/ItemIdSearchTextarea.vue:102]: This one was left over from the token replacement

File: resources/js/Pages/Layout.vue [hunk 2]
```diff
@@ -190,15 +190,16 @@ onBeforeUnmount(() => {
...
 @import '../../sass/typography';
+@import '../../css/custom-variables.css';
 
 #app {
     box-sizing: border-box;
     min-height: 100%;
     display: flex;
     flex-direction: column;
-
+
```
Comment (guergana) [inline resources/js/Pages/Layout.vue:200]: :grimacing:

File: resources/js/Pages/Home.vue [hunk 3]
```diff
@@ -189,6 +189,7 @@ function showRandom(): void{
     display: flex;
     justify-content: space-between;
     align-items: center;
+    gap: var(--dimension-spacing-small);
```
Comment (guergana) [inline resources/js/Pages/Home.vue:192]: The gap is for this spec, right? 
image

**Review by guergana — COMMENTED** [review]
Just left two questions. Other than that it looks good.

File: resources/js/Pages/Home.vue (hunk 3 above)
Comment (chukarave) [reply resources/js/Pages/Home.vue:192]: For this part: 
image

But I realized I skipped the gap in `Results.vue` so I should add that too

File: resources/js/Pages/Layout.vue (hunk 2 above)
Comment (guergana) [reply resources/js/Pages/Layout.vue:200]: There's this very cool plugin you can install that will get rid of that white space: https://marketplace.visualstudio.com/items?itemName=EditorConfig.EditorConfig

**Review by itamargiv — APPROVED** [review]
The css looks alright to me, I think we can leave the whitespace issue to another time. Thank you!

**Pull request merged**